documentation/specs/src/interoperability/ethereum-bridge.md

```diff
@@ -13,13 +13,24 @@ The Namada Ethereum bridge system consists of:
   [ICS20](https://docs.cosmos.network/v0.42/modules/ibc/) fungible token 
   transfers.
 * A set of Ethereum smart contracts.
-* A relayer for submitting transactions to Ethereum.
+* An automated process to send validator set updates to the Ethereum smart 
+  contracts.
```

Comment:
should we mention that without vote extensions this process may fallback to a governance vote?

Reply:
as in, we reach the end of an epoch without getting >2/3 of voting power on a validator set update certificate (comprised of signatures from a quorum of validators)

Reply:
Oh, I do remember. Just had to start typing up the details to realize it.. Still coffeeing up. :sweat_smile:

Reply:
Actually, I think we should not use governance for this.